### PR TITLE
Add wandb resume argument

### PR DESCRIPTION
**Pull request by drkostas**
https://docs.wandb.ai/guides/runs/resuming

For graph consistency

### CODE REVIEW

**Review by andreicozma1 — CHANGES_REQUESTED** [review]

File: main_pretrain.py [hunk 1]
```diff
@@ -249,6 +249,13 @@ def get_args_parser():
         default=None,
         help="The path to the checkpoint to resume training from.",
     )
+    # https://docs.wandb.ai/guides/runs/resuming
+    parser.add_argument(
+        "--wandb_resume",
+        type=str,
+        default=None,
+        help="The wandb projects name to resume training from.",
+    )
```
[the remark this answers is not in the document]
Comment (andreicozma1) [reply main_pretrain.py:258]: oh didn't see the PR marked as draft till I refreshed the page

[the remark this answers is not in the document]
Comment (drkostas) [reply main_pretrain.py:258]: Haha yes still working on it

File: main_pretrain.py [hunk 2]
```diff
@@ -443,16 +458,42 @@ def main(args):
...
                 group=args.model,
                 job_type="pretrain",
+                resume=None if args.resume is None else "must",
+                id=wandb_id,
             )
-            wandb.config.update(args)
+            wandb_args = copy(args)
+            if args.resume is not None:
+                wandb_args.start_epoch += 1
+            wandb.config.update(wandb_args, allow_val_change=True)
```
Comment (andreicozma1) [inline main_pretrain.py:493]: nice!

**Review by andreicozma1 — APPROVED** [review]
All looks good! no questions from my side

**Review by maofengggg — APPROVED** [review]
coool!

**Pull request merged**